.eslintrc

```diff
@@ -5,52 +5,41 @@
     "mocha": true,                   // mocha keywords
     "es6": true
   },
+  "plugins": [
+    "prettier"
+  ],
+  "extends": [
+    "prettier"
+  ],
   "rules": {
```

Comment:
I'll explain myself. Almost every linter, including eslint, have two kind of rules:
- Formatting rules: These restrict the way we write code, for example: indent, arrow-spacing, max-len, etc.
- Code quality rules: These rules restrict the way we organize and execute code, for example: complexity, default-case, constructor-super, etc.

Formatting rules enforce code style consistency while code quality rules enforce good practices and avoids anti-patterns.

In my previous comment I suggest leaving all the setup of formatting rules to prettier, which is a code formatter that has an opinionated and widely approved standard. We still need to take care of code quality rules, my suggestion here is using another standard that's widely approved by the community, which is airbnb's javascript guide.